UnityProject/Assets/Scripts/ScriptableObjects/Learning/TipsSO.cs

```diff
@@ -0,0 +1,9 @@
+﻿using System.Collections.Generic;
+using UnityEngine;
+
+
+[CreateAssetMenu(fileName = "Tips", menuName = "Learning/Data/Tips")]
+public class TipsSO : ScriptableObject
```

Comment:
~[developers devel](https://www.youtube.com/watch?v=KMU0tzLwhbE)~ err... `namespaces`, `namespaces`, `namespaces`...

Reply:
Actually, instead of creating a new SO type for such a generic usage, use `StringList` instead.